test/node-rpc-test.js

```diff
@@ -236,6 +236,90 @@ describe('RPC', function() {
     });
   });
 
+  describe('getblockchaininfo', function() {
+    const check = (info, options) => {
+      assert.ok(info);
```

Comment:
nit: `assert()` === `assert.ok()`, shorter to use `assert()` and more consistent.

Reply:
yeah, but `.ok` kinda felt for testing was more descriptive. That's the only reason for `.ok`s.
But can go back to `assert`s.